tmessage/cli.py

```diff
@@ -19,9 +19,13 @@
 parser.add_argument('--user', action='store', type=str, required=True)
 parser.add_argument('--server', action='store', type=str)
 parser.add_argument('--port', action='store', type=int)
+parser.add_argument('--store',
+                    action='store_true',
+                    help='Store messages in JSON format.')
 
 args = parser.parse_args()
 
+IS_STORE = args.store

```

Comment:
@Dansyuqri Make the default value of this variable to be `True`.

Reply:
Hi @Haider8 , the default boolean value of `True` is initialized in the preceding line 
```python
parser.add_argument('--store',
                    action='store_true',
                    help='Store messages in JSON format.')
```
in the `action` param. Is this what you're referring to?

Reply:
But it is not storing the messages when `--store` is not passed.

Reply:
`IS_STORE = args.store or True`
Will this help?

Reply:
Just to clarify, the messages should be stored only when `--store` argument is passed in CLI right? And when `--store` is not passed, the messages should not be stored?

Reply:
No. Messages should be stored when `--store` is not passed (the default behavior).

Reply:
Storing of messages is the feature we want but if the user doesn't want to store he/she can turn that off. Simple

Reply:
And it should not store when `--store` is passed into the cli?

Sorry for asking a lot, but I want to get it right, thank you.

Reply:
Okay, noted on this. Thank you

Reply:
It's getting confusing now. Change the variable name to `--store_off` from `--store`. The current approach will then work fine. Make the other changes in variable names (if required).

Reply:
we can Try implementing like this @Haider8 @Dansyuqri
```
parser.add_argument('--store', dest='store', action='store_true')
parser.add_argument('--dont-store', dest='store', action='store_false')
parser.set_defaults(store=True)
```
Messages will be stored when we pass --store or not
but when user doesn't want to store messages he need to pass --dont-store

Reply:
Could I suggest a change in the cli param to be `--no-store` instead? so that it is more intuitive for the user, since to turn it off, the `--no-store` param can be used

Reply:
@Haider8 @Shanmugapriya03 Definitely, thank you for your patience. Will implement and make the changes accordingly with the docs.